node_modules/oae-principals/emailTemplates/invite.html.jst

```diff
@@ -0,0 +1,195 @@
+<%
+    /**
+     * Render the email header
```

Comment:
Seems like there is a lot of duplication going on between this template and the notifications template